src/commands/job/spec/jobCommand.spec.ts

```diff
@@ -0,0 +1,322 @@
+import path from 'path'
+import * as executeModule from '../execute'
+import { JobCommand } from '../jobCommand'
+import { AuthConfig, Logger, LogLevel, ServerType, Target } from '@sasjs/utils'
+import * as configUtils from '../../../utils/config'
+import { ReturnCode } from '../../../types/command'
+import { setConstants } from '../../../utils'
+
+const defaultArgs = ['node', 'sasjs']
+const target = new Target({
+  name: 'test',
+  appLoc: '/Public/test/',
+  serverType: ServerType.SasViya,
+  contextName: 'test context'
+})
+const authConfig = {}
+const jobFileName = 'someJob'
+const jobPath = `/Public/folder/${jobFileName}`
+const log = './jobLog.log'
+const output = './jobOutput.json'
+const statusFile = './status/file.txt'
+const source = './macros.json'
+const projectFolder = __dirname
+describe('JobCommand', () => {
+  beforeAll(async () => {
+    process.projectDir = projectFolder
+  })
+
+  beforeEach(() => {
+    setupMocks()
+  })
+
+  it('should parse sasjs job execute command', async () => {
+    const args = [...defaultArgs, 'job', 'execute', jobPath]
+
+    const command = new JobCommand(args)
```

Comment:
would be nice to put it in function to reduce code duplication

Reply:
refactored specs, fixed.